### ISSUE TITLE
Can't add contract to function with lang item attribute

### ISSUE THREAD

**Issue by carolynzech** [model-checking/kani#3427]
I tried this code:
```rust
#[lang = "align_offset"]
#[safety::requires(a.is_power_of_two())]
pub(crate) const unsafe fn align_offset<T: Sized>(p: *const T, a: usize) -> usize {
```

with Kani version: 0.5.3

I expected to see this happen: verification succeeded

Instead, this happened: 

```
error[E0152]: found duplicate lang item `align_offset`
    --> /Users/cmzech/personal-verify-std/library/core/src/ptr/mod.rs:1812:1
     |
1812 | #[safety::requires(a.is_power_of_two())]
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
     |
note: the lang item is first defined here
    --> /Users/cmzech/personal-verify-std/library/core/src/ptr/mod.rs:1812:1
     |
1812 | #[safety::requires(a.is_power_of_two())]
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
     = note: this error originates in the attribute macro `kani_core::requires` (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to 1 previous error
```
This isn't customer-facing, and once can always verify a copy of the function without the lang-item, so not necessarily high priority.

**Comment by artemagvanian** [comment]
The reason for that could be that multiple bodies are generated for each function and attributes are copied for all of them. I think it should be possible to filter the attributes here: https://github.com/model-checking/kani/blob/e2a209bcb847803e8a5abb20ef7002b109ed0bf6/library/kani_macros/src/sysroot/contracts/bootstrap.rs#L33-L47

**Comment by carolynzech** [comment]
Fixed in Kani v0.5.4
